Catappult/appcoins-wallet-android
# [APPC-2273] - Change TopApps url.

**What does this PR do?**

Change TopApps url to https://en.aptoide.com/store/catappult/group/group-10867.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] TransactionsViewModel.java

**How should this be manually tested?**

  Open the wallet with no transactions. In Transactions screen view pager, swipe if needed to see "Discover Top Apps". When clicking the link you should now be directed to https://en.aptoide.com/store/catappult/group/group-10867 instead of the old link https://en.aptoide.com/store/bds-store/group/group-10867.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-2273](https://aptoide.atlassian.net/browse/APPC-2273)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)

No


**Code Review Checklist**
